features/keychain/module/keychain.js

```diff
@@ -25,6 +25,8 @@ export const MODULE_ID = 'keychain'
 export class Keychain {
   #masters = Object.create(null)
   #legacyPrivToPub = null
+  #lockedPrivateKeys = false
+  #getPrivateHDKeySymbol = Symbol('getPrivateHDKey')
```

Comment:
Is this a problem? We don't need it per-se, but mostly we only have 1 keychain instance right? What is the downside?

Reply:
no downside, just wondering if it was intentional. In the past we've usually used a single Symbol instance, but i can see how it might be better to use one for instance in the case of keychains

Reply:
It was intentional, the purpose was that the method with `getPrivateHDKeySymbol` can only be called internally to the instance

Reply:
I am lacking imagination of how this could be abused from the outside if we declare a non-exported symbol as const and not class property 😄

 2 reasons:
- consumers can't get their hands on the symbol if it's not exported
- the parameter is not exposed to consumers as it is a private method that accepts it, so even if somehow a consumer could get hold of the symbol, they can't pass it in

Reply:
Yeah, seems unlikely. Happy to move it out if preferred, just wanted to be extra safe for the keychain

Reply:
that would be my preference as it makes the code a slightly less clear (as a reader I keep wondering why does this have to be a class prop?) but just a tiny tiny nit, so feel free to ignore